release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -69,7 +69,8 @@ before booting a VM from an OVA in vSphere:
 
 [source,terminal]
 ----
-$ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=<static_ip_config>"
+$ export IPCFG="ip=10.20.30.42::10.20.30.254:255.255.255.0:mynode01:ens192:off"
+$ govc vm.change -vm "${VM_NAME}" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
```

Comment:
The suggested snippet was actually from a larger set of commands that can be seen at https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-vmware/#_first_boot_networking_and_ignition

Maybe just replace `"${VM_NAME}"` with an obviously fake test name or something similar that indicates they need to replace it with their own value.

Reply:
Similar to what we've indicated in this article - https://access.redhat.com/solutions/4175151 - I would add the nameserver entries and give maybe give 1 example:

ip=<ip>::<gateway>:<netmask>:<hostname>:<iface>:none nameserver=srv1 [nameserver=srv2 [nameserver=srv3 […]]

Reply:
@syangsao Made those updates in 0e6f4b8287d3d684205d44165bde9ed5f81b760a. Can you verify?